src/ocean/util/app/ext/LogExt.d

```diff
@@ -79,22 +79,38 @@ class LogExt : IConfigExtExtension
     public bool use_insert_appender;
 
 
+    /// If set, will be used by `makeLayout` to map a name to a `Layout`
+    private Appender.Layout delegate (cstring name) layout_maker;
+
+
     /***************************************************************************
 
         Constructor.
 
         Params:
+            make_layout         = A delegate that instantiate an
+                                  `Appender.Layout` from a name. If null,
+                                   defaults to `ocean.util.Config: newLayout`.
             use_insert_appender = true if the InsertConsole appender should be
                                   used (needed when using the AppStatus module)
 
     ***************************************************************************/
 
-    this ( bool use_insert_appender = false )
+    this ( bool use_insert_appender )
     {
-        this.use_insert_appender = use_insert_appender;
+        this(null, use_insert_appender);
     }
 
 
+    /// Ditto
+    public this ( Appender.Layout delegate (cstring name) make_layout = null,
+                 bool use_insert_appender = false)
```

Comment:
The code style of the ctor signature doesn't seem to be consistent. Either you keep space for open/closed parenthesis or you remove for both of them